src/constants/defaultTheme.ts

```diff
@@ -0,0 +1,3 @@
+export default {
+  mobileWidth: 560,
```

Comment:
There was two `defaultTheme.ts` files, I only moved one of them since I thought maybe we don't need it. 
Now I copied it as well and tried to make the theme components as close as ConnectKit's original code.